docs/excel/custom-functions-parameter-options.md

```diff
@@ -28,31 +27,83 @@ Whereas regular parameters are required, optional parameters are not. When a use
  * @returns {number} The sum of the numbers.
  */
 function add(first, second, third) {
-  if (third !== undefined) {
-    return first + second + third;
+   if (third === null) {
```

Comment:
Need to fix indentation of this line (`if...`) remove the extra space at the beginning of the line.